source/_integrations/bluetooth.markdown

```diff
@@ -87,6 +87,25 @@ The following methods are known to work to add multiple adapters:
 
 Integrations that have followed the [Best practices for library authors](https://developers.home-assistant.io/docs/network_discovery?_highlight=bluetooth#best-practices-for-library-authors) will automatically connect via the adapter with the best signal and failover to an active adapter if one becomes unavailable.
 
+## Remote adapters
+
+The Bluetooth integration supports receiving advertisement data from external adapters for devices and sensors that do not need an active connection. The number of remote scanners is limited only by the performance of the host system.
+
+The following remote adapters are supported:
+
+- ESPHome
+
+### ESPHome requirements
+
+Devices with an ESP32 chip running [ESPHome](/integrations/esphome/) must enable the `bluetooth_proxy` component and be added to Home Assistant before advertisements are forwarded.
```

Comment:
When there is documentation on the esphome site about this we can likely collapse this section, but I didn't want to roll out beta without it since I'm already getting questions about HOWTO

Reply:
@jesserockz FYI. Can you update this doc once the ESPHome docs are live?